### ISSUE TITLE
Detected by familysearch.org

### ISSUE THREAD

**Issue by alexreg** [ultrafunkamsterdam/undetected-chromedriver#1018]
Chrome Version 109.0.5414.119 (Official Build) (arm64)
UC 3.2.2
OS: macOS Ventura 13.1

I get the Incapsula "access denied" page. Is this an issue with the library, or am I doing something obviously wrong?

```python
        driver.get("https://www.familysearch.org")

        accept_button: uc.WebElement = WebDriverWait(driver, timeout=10).until(ec.visibility_of_element_located((By.ID, "truste-consent-button")))
        accept_button.click_safe()

        signin_button: uc.WebElement = WebDriverWait(driver, timeout=10).until(ec.visibility_of_element_located((By.ID, "signInLink")))
        signin_button.click_safe()
```

**Comment by HkerVit** [comment]
i access https://www.familysearch.org normal. Try new IP ?

**Comment by alexreg** [comment]
@HkerVit Thanks for trying. New IP doesn't seem to help. I presume you ran by above code snippet? I always get "Access Denied, Error 15" after the sign-in page loads.

**Comment by lukect** [comment]
Fixed: #1010

**Comment by alexreg** [comment]
@lukect Thank you, great work.
